### ISSUE TITLE
Issue with DateTime type for AbstractOrder::$terms and AbstractOrder::$privacy

### ISSUE THREAD

**Issue by ocean90** [concardis/PHP_SDK#3]
The code documentation says that both properties accept a DateTime object but the API only accepts a timestamp (integer) in requests.

Request:
<img width="778" alt="request" src="https://user-images.githubusercontent.com/617637/32059605-ea76fab6-ba6c-11e7-9353-ce7387579bab.png">

Response:
<img width="388" alt="response" src="https://user-images.githubusercontent.com/617637/32059622-f25583d8-ba6c-11e7-82b4-0e0f2a9f41cb.png">

Example of a request payload that produces a "Malformed request!" error.
```json
{
   "payment":{
      "paymentInstrumentId":"paymentinstrument_xxxxxxxx"
   },
   "product":"creditcard",
   "async":{
      "successUrl":"http:\/\/google.de?q=success",
      "failureUrl":"http:\/\/google.de?q=failure",
      "cancelUrl":"http:\/\/google.de?q=cancel"
   },
   "terms":{
      "date":"2017-10-26 14:48:15.000000",
      "timezone_type":3,
      "timezone":"UTC"
   },
   "privacy":{
      "date":"2017-10-26 14:48:15.000000",
      "timezone_type":3,
      "timezone":"UTC"
   },
   "description":"Tickets",
   "initialAmount":6000,
   "currency":"CHF",
   "basket":[
      {
         "name":"Stehplatz",
         "totalPrice":6000,
         "unitPrice":6000,
         "quantity":1
      }
   ],
   "customer":"customer_xxxxxxxx",
   "channel":"ECOM",
   "source":"Website"
}
```

**Comment by ocean90** [comment]
Turns out the middleware requires the use of milliseconds. To get the current time in milliseconds you have to use `round( microtime( true ) * 1000 );` which will return a float by default.

**Comment by Sharenkamp** [comment]
Hi @ocean90,

the upcoming release will support timestamp and Datetime.
The conversion to milliseconds will be handled by the SDK.

For the current version your solution works fine.

best regards,
Stefan

**Comment by Sharenkamp** [comment]
Fixed with 1.1.0
